src/rpcrawtransaction.cpp

```diff
@@ -434,6 +434,7 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
             "      ,...\n"
             "    }\n"
             "3. locktime                (numeric, optional, default=0) Raw locktime. Non-0 value also locktime-activates inputs\n"
+            "4. expiryheight            (numeric, optional, default=0) Expiry height of transaction\n"
```

Comment:
If I were being pedantic I would suggest adding "(if Overwinter is active)", but since it already is on mainnet, that isn't needed.